src/network/networkd-sysctl.c

```diff
@@ -315,11 +376,44 @@ static int link_set_ipv4_promote_secondaries(Link *link) {
         return sysctl_write_ip_property_boolean(AF_INET, link->ifname, "promote_secondaries", true, &link->sysctl_shadow);
 }
 
+static void link_watch_sysctl(Link *link) {
+        sd_event_source **source;
+        int r;
+
+        assert(link);
+
+        source = link->sysctl_event_sources;
+
+        FOREACH_STRING(p,
+                "/proc/sys/net/ipv4/conf",
+                "/proc/sys/net/ipv6/conf") {
+                _cleanup_free_ char *path = path_join(p, link->ifname);
+
+                if (!path)
```

Comment:
we usually want to keep function calls and their immediate error handling together. i.e. place empty lines before a function call line, and/or after the error handling, but do not separate the function call from its error handling. or in other words, just remove the empty line before this one

Reply:
But I also see that we leave a space between the variable declaration and the code, in this case, which rule prevails?